From 1228dfab878f0a31b470f4a5e944075432fc0394 Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Sun, 14 Aug 2016 11:23:20 +0800
Subject: [PATCH 7/7] init: enlarge the time to wait coldboot done

We need more time to probe and load all modules.
If 10s is still not enough for you, let me know.
---
 init/init.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init/init.cpp b/init/init.cpp
index 83104ca..7da06d7 100644
--- a/init/init.cpp
+++ b/init/init.cpp
@@ -163,10 +163,10 @@ static int wait_for_coldboot_done_action(const std::vector<std::string>& args) {
     Timer t;

     NOTICE("Waiting for %s...\n", COLDBOOT_DONE);
-    // Any longer than 1s is an unreasonable length of time to delay booting.
+    // Any longer than 10s is an unreasonable length of time to delay booting.
     // If you're hitting this timeout, check that you didn't make your
     // sepolicy regular expressions too expensive (http://b/19899875).
-    if (wait_for_file(COLDBOOT_DONE, 1)) {
+    if (wait_for_file(COLDBOOT_DONE, 10)) {
         ERROR("Timed out waiting for %s\n", COLDBOOT_DONE);
     }

--
2.7.4
